build/deployments/docker-compose.yaml

```diff
@@ -95,3 +95,40 @@ services:
   # Uncomment to enable DLV debugging
   # environment:
   #   - DEBUG_PORT=7084
+
+  db:
+    container_name: pocket-db
+    # image: postgis/postgis  # TODO(olshansky): switch image when we start adding support for GeoZones
+    image: postgres:14.1-alpine
+    restart: always
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - "5432:5432"
+    volumes:
+      - ${PWD}/build/sql:/tmp/scripts
+      - db:/var/lib/postgresql/data
+
+  pgadmin:
+    container_name: pgadmin
+    image: dpage/pgadmin4
+    restart: always
+    depends_on:
+      - db
+    environment:
+      PGADMIN_DEFAULT_EMAIL: pgadmin4@pgadmin.org
```

Comment:
Pgadmin4 sees like it might be useful, but there's no documentation for me to use. Consider adding one. I will approve the changes either way

Reply:
@andrewnguyen22  Done. It's very rudimentary though.